pkg/apis/core/v1beta1/conversions.go

```diff
@@ -66,6 +66,20 @@ func addConversionFuncs(scheme *runtime.Scheme) error {
 		return err
 	}
 
+	if err := scheme.AddFieldLabelConversionFunc(
+		SchemeGroupVersion.WithKind("Project"),
+		func(label, value string) (string, string, error) {
+			switch label {
+			case "metadata.name", "metadata.namespace", core.ProjectNamespace:
```

Comment:
Can we omit `metadata.namespace` as `Project`s are not namespaced (see `ControllerInstallation`)?